Libraries/Image/resolveAssetSource.js

```diff
@@ -25,6 +25,7 @@ var AssetRegistry = require('AssetRegistry');
 var PixelRatio = require('PixelRatio');
 var Platform = require('Platform');
 var SourceCode = require('NativeModules').SourceCode;
+var assetPathUtils = require('../../local-cli/bundle/assetPathUtils');
```

Comment:
Since both resolveAssetSource.js and getAssetDestPath.js is going to depend on this refactor, extracting this change is going to be hard and confusing because then one PR would rely on the other PR getting merged.